src/api/app/helpers/webui/buildstatus_helper.rb

```diff
@@ -0,0 +1,63 @@
+module Webui::BuildstatusHelper
+  # rubocop:disable Style/AlignHash
```

Comment:
Rubocop complains about the indentation. I talked to several developers here and no one could really help me. The only solution what I found was "disable it". It seems this is a bug in rubocop.